packages/eslint-plugin/rules/dependency-group.js

```diff
@@ -1,7 +1,8 @@
 /** @typedef {import('estree').Comment} Comment */
 /** @typedef {import('estree').Node} Node */
 
-module.exports = /** @type {import('eslint').Rule.RuleModule} */ ( {
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
```

Comment:
Minor tweak, this is a type annotation instead of a type assertion. This should be safer.

Reply:
Could you elaborate? Safer how? 🙇🏻

Reply:
It may or may not actually be safer in this case, but it is the standard way of declaring a type and should be weaker than the assertion (weaker being better here in my estimation, let the type system do more of the work). This is equivalent to the [standard method of declaring a type covered here](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html). Before it was a [type assertion](https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions):

> Type assertions are a way to tell the compiler “trust me, I know what I’m doing.” A type assertion is like a type cast in other languages, but it performs no special checking or restructuring of data. It has no runtime impact and is used purely by the compiler. TypeScript assumes that you, the programmer, have performed any special checks that you need.